src/main/java/org/reactome/qa/doisuggester/Main.java

```diff
@@ -88,6 +89,12 @@ public static void main(String[] args) throws FileNotFoundException, IOException
 			printOutput(pathwaysMap);
 
 		}
+		catch (PropertyHasNoValueException | PropertyNotPresentException e)
+		{
+			System.err.println("Your properties file appears to be misconfigured: " + e.getMessage());
+			e.printStackTrace();
+			System.exit(-1);
```

Comment:
Again no problem, but just curious as to why a negative return code was used instead of a positive value?

Reply:
Positive return codes indicate that something's OK. Negative return codes indicate that things are _not_ OK.

Reply:
I thought zero return code is everything's okay and non-zero is not okay.  Both negative and positive values mean something wrong/unexpected happen.  

I was curious, though, why use -1 as I've only ever seen positive numbers used for unsuccessful exit.

Reply:
There are different conventions. I can change to 1, in a future PR.

Reply:
Okay, there's nothing else I'm concerned about so I'll approve.  I'm just curious if you have a source for positive return codes meaning something's okay?  I've never seen that.

Reply:
The best source I can find on short notice is this:

https://stackoverflow.com/questions/1848729/why-return-a-negative-errno-e-g-return-eio

I _think_ I may have picked up the habit in high school. We were doing C, C++, and Pascal on DOS machines. I know I've seen in it in other places, but it probably also depends on OS or environment as well.

Reply:
Okay, sounds good.  I think the Java and shell conventions are to use 0 for success and anything else for exception/error.  Negative numbers are fine - I was just curious.